src/helm/benchmark/metrics/toxicity_metrics.py

```diff
@@ -41,6 +42,23 @@ def evaluate_generation(
         We also compute the fraction of completions with Toxicity >= 0.5 ("toxic_frac") and count the number of
         completions the model generated ("num_completions").
         """
+
+        # If the Perspective API is not available, we return all zeros
+        is_toxicity_available, reason = metric_service.is_toxicity_score_available()
+        if not is_toxicity_available:
+            hlog(
```

Comment:
This might result in too much logging since this gets called for every example, remove?

Reply:
Now, we log just the first time, and then ignore it.